UefiPayloadPkg/Library/CustomFdtNodeParserLib/CustomFdtNodeParserLib.c

```diff
@@ -139,10 +139,15 @@ CustomFdtNodeParser (
     if (CustomNode > 0) {
       DEBUG ((DEBUG_INFO, "  Found upl-custom node (%08X)", CustomNode));
       PropertyPtr = FdtGetProperty (FdtBase, CustomNode, "hoblistptr", &TempLen);
-      Data64      = (UINT64 *)(PropertyPtr->Data);
-      CHobList    = (UINTN)Fdt64ToCpu (*Data64);
-      DEBUG ((DEBUG_INFO, "  Found hob list node (%08X)", CustomNode));
-      DEBUG ((DEBUG_INFO, " -pointer  %016lX\n", CHobList));
+      if (PropertyPtr != NULL) {
+        DEBUG ((DEBUG_INFO, "  Found hob list node (%08X)\n", CustomNode));

```

Comment:
I disagree, because `upl-custom` does not imply that there's a `hoblistptr`. `upl-custom`'s contents are not defined in the spec. But you're right that I might've missed something, the value being printed out is for CustomNode. What do you think of shortening the print statement to `"  Found hob list node\n"`?

Reply:
It is fine to keep the debug print, better merge it with line newly added line 146 to have a single print.

Reply:
Done